services/QuillLMS/spec/controllers/auth/classlink_controller_spec.rb

```diff
@@ -22,8 +22,8 @@ def setup_conflicting_user_response(user)
     subject { get :classlink, params: { code: } }
 
     before do
-      allow(ClasslinkIntegration::AuthCredentialSaver).to receive(:run)
-        .and_return(auth_credential)
+      allow(ClasslinkIntegration::AuthCredentialSaver).to receive(:run).and_return(auth_credential)
+      allow(controller).to receive(:sign_in)
```

Comment:
This change shouldn't be necessary since there were only whitespace edits to `ClasslinkController`